native/cocos/bindings/manual/jsb_global.cpp

```diff
@@ -1526,7 +1526,7 @@ SE_BIND_FUNC(JSB_openharmony_postSyncMessage)
 #endif
 
 bool jsb_register_global_variables(se::Object *global) { // NOLINT
-    gThreadPool = LegacyThreadPool::newFixedThreadPool(3);
+    gIOThreadPool = LegacyThreadPool::newFixedThreadPool(5);
```

Comment:
There may be lots of json/ccb files to be loaded, so add 2 more threads in pool.